easybuild/easyconfigs/l/libxc/libxc-3.0.0-foss-2016b.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'ConfigureMake'
+
+name = 'libxc'
+version = '3.0.0'
+
+homepage = 'http://www.tddft.org/programs/octopus/wiki/index.php/Libxc'
+description = """Libxc is a library of exchange-correlation functionals for density-functional theory.
+ The aim is to provide a portable, well tested and reliable set of exchange and correlation functionals."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+sources = [SOURCE_TAR_GZ]
+source_urls = ['http://www.tddft.org/programs/octopus/down.php?file=libxc/']
+
+configopts = 'FC="$F77" FCFLAGS="$FFLAGS" --enable-shared --enable-fortran'
+
+# From the libxc mailing list
+# To summarize: expect less tests to fail in libxc 2.0.2, but don't expect
+# a fully working testsuite soon (unless someone wants to volunteer to do
+# it, of course  ) In the meantime, unless the majority of the tests
+# fail, your build should be fine.
+runtest = 'check'
```

Comment:
sorry, my comment "see #4397" was too laconic, let me fix that :) 

In #4397, we discuss the origin of those failures in Haswell only, and suggest disabling `optarch` for `libxc`, since the tests always pass without `optarch`. I'm of the opinion that it is more important for libxc to be reproducible than to be fast (and I doubt that libxc has much impact on the overwall execution times of the application using it, much less any eventual performance difference from the optimization flags of libxc alone, unless you're doing something very specific with the functionals themselves, in which case there could be a performance difference but it would still be more important to be reproducible), so I'd disable `optarch` and enable the tests for all libxc easyconfigs...

from the libxc homepage: "The aim is to provide a portable, well tested and reliable set of exchange and correlation functionals"

of course, one could argue that there is not exact reproducibility with floating point arithmetic, but I believe the libcx tests have already been relaxed to take that into account in version 2.1.0, this should be a different problem

too verbose? :)